pyaedt/edb_core/edb_data/siwave_simulation_setup_data.py

```diff
@@ -365,12 +367,23 @@ def __init__(self, parent):
     def sim_setup_info(self):
         """EDB internal simulation setup object."""
 
-        return self._parent._edb_sim_setup_info
+        return self._parent._edb_object
 
     @property
     def min_void_area(self):
         """Minimum area below which voids are ignored.
 
+        Returns
+        -------
+        float
+        """
+        warnings.warn("`min_void_area` is deprecated. Use `dc_min_void_area` property instead.", DeprecationWarning)
+        return self.dc_min_void_area_to_mesh
+
+    @property
+    def dc_min_void_area_to_mesh(self):
+        """Minimum area below which voids are ignored.
```

Comment:
Shouldn't DC somehow be in this description to distinguish this property from the previous one?